tests/_support/Helper/crud-helpers/order.php

```diff
@@ -295,12 +295,14 @@ public function print_query( $id ) {
 			'isDownloadPermitted'   => $data->is_download_permitted(),
 			'needsShippingAddress'  => $data->needs_shipping_address(),
 			'hasDownloadableItem'   => $data->has_downloadable_item(),
-			'downloadableItems'     => array_map(
-				function( $download ) {
-					return array( 'downloadId' => $download->get_id() );
-				},
-				$data->get_downloadable_items()
-			),
+			'downloadableItems'     => ! empty( $data->get_downloadable_items() )
```

Comment:
I'd recommend to refactor this part to make easier for newcomers to read and understand.

Reply:
The helper classes need heavy refactoring and documentation. This should get a issue/PR of its own.